package.json

```diff
@@ -22,6 +22,7 @@
   },
   "homepage": "https://github.com/maykinmedia/django-rijkshuisstijl#readme",
   "dependencies": {
+    "@babel/compat-data": "^7.9.0",
```

Comment:
Waarom is deze nodig?

Reply:
Inmiddels heb ik vanmiddag een reboot gedaan (dus geen logs ervan) maar dat zonder die lib kon gulp niet builden